app/controllers/avo/base_controller.rb

```diff
@@ -570,10 +582,6 @@ def set_pagy_locale
       @pagy_locale = locale.to_s || Avo.configuration.default_locale || "en"
     end
 
-    def safe_call(method)
```

Comment:
Don't we need this? And shouldn't it take more args?

Reply:
I guess you missed this change https://github.com/avo-hq/avo/pull/2703/files#diff-27ec89c6677b158fe3a800b268c9c196b5409f439315240504626f4c8bf81bc7R140

The method was extracted to be accessible in more places